### PR TITLE
docs(validators): apply i18n for global

### PR DESCRIPTION
**Pull request by ChanhyukPark-Tech**
## Overview

I have translated the @toss/validators package to the i18n specification.

### Question

Currently, the folder structure seems to be not clean because there are implementation functions, test codes, and document files in one source file.

image

It may be okay now, but I think the bigger the @toss/validators package gets, the more messy it gets.

Is there a reason why you designed it like this? Here's how I thought about it.

```
📦src
 ┣ 📂validators
 ┃ ┗ 📜is-age.ts
 ┃ ┗ 📜is-email.ts
 ┣ 📂__tests__
 ┃ ┗ 📜is-age.spec.ts
 ┃ ┗ 📜is-email.spec.ts
 ┣ 📂docs
   ┗ 📜is-age.ko.md
   ┗ 📜is-age.en.md
   ┗ 📜is-email.ko.md
   ┗ 📜is-email.en.md
```

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/common/validators/src/validators/is-email.ko.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+```typescript
+function isEmail(email: string): boolean;
+```
+
+## Examples
+
+```typescript
+isEmail('raon0211@gmail.com'); // true
+isEmail('raon0211@gmail.'); // false
+isEmail('raon0211@gmail.com123'); // false
```
Comment (ChanhyukPark-Tech) [inline packages/common/validators/src/validators/is-email.ko.md:18]: 이 부분에서 `raon0211` 이라는 키워드를 사용했기때문에 raon0211님께서 괜찮으신지 확인해주셨으면 합니다. 만약 문제가 있다면 바로 다른 키워드로 수정하겠습니다!

**Comment by raon0211** [comment]
It was by intention that the source code, test code, and documentation are in the same directory. I thought they will always be edited together, so putting them as near as possible would be a better choice. (If they are far apart, we may forget to edit some files)

But I agree that it might look messy as more files are added. For that case, what about arranging files like this?

```
is-age/
  index.ts
  index.spec.ts
  index.en.md
  index.ko.md
is-birth-date-6/
  index.ts
  index.spec.ts
  index.en.md
  index.ko.md
```

File: packages/common/validators/src/validators/is-age-en.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: is-age
```
Comment (raon0211) [inline packages/common/validators/src/validators/is-age-en.md:2]: We plan to match the documentation’s title with the actual function name. What about changing the title to `isAge`? (And other docs accordingly)

File: packages/common/validators/src/validators/is-age.ts [hunk 3]
```diff
@@ -1,17 +1,3 @@
...
- * ```typescript
- * function isAge(ageInput: string): boolean;
- * ```
- *
- * @example
- * isAge('25') // true
- * isAge('asd') // false
- * isAge('24.3') // false
- */
 export function isAge(ageInput: string): boolean {
```
Comment (raon0211) [inline packages/common/validators/src/validators/is-age.ts:1]: For now we have to add tossdocs-ignore to each of the source file 😢 (After migrating all of our docs out of jsdoc, we will remove all tossdocs-ignore

File: packages/common/validators/src/validators/is-email.ko.md (hunk 1 above)
Comment (raon0211) [reply packages/common/validators/src/validators/is-email.ko.md:18]: Oh.. it was originally our internal docs, but I think setting this to some anonymous email (`johndoe@example.com`) would be better 😄

Comment (ChanhyukPark-Tech) [reply packages/common/validators/src/validators/is-email.ko.md]: thank you! I fixed it

Comment (ChanhyukPark-Tech) [reply packages/common/validators/src/validators/is-email.ko.md]: [fix: is-email example personal email => anonymous email](https://github.com/toss/slash/pull/56/commits/e495dfb66a0b04fa4bccd7d26627b05e826748f2)
[e495dfb](https://github.com/toss/slash/pull/56/commits/e495dfb66a0b04fa4bccd7d26627b05e826748f2)

File: packages/common/validators/src/validators/is-age.ts (hunk 3 above)
Comment (ChanhyukPark-Tech) [reply packages/common/validators/src/validators/is-age.ts:2]: yes! thank you
[fix: write /** @tossdocs-ignore */ plan to remove after migrate](https://github.com/toss/slash/pull/56/commits/7f66ba1ffd3898347e9b14b0d263d5f80d99cde1)
[7f66ba1](https://github.com/toss/slash/pull/56/commits/7f66ba1ffd3898347e9b14b0d263d5f80d99cde1)

File: packages/common/validators/src/validators/is-age-en.md (hunk 2 above)
Comment (ChanhyukPark-Tech) [reply packages/common/validators/src/validators/is-age-en.md]: [fix: match documentations't title with actual function name](https://github.com/toss/slash/pull/56/commits/589cdba91ca4c8e80cc6c3c2094315d9a8239d43)
[589cdba](https://github.com/toss/slash/pull/56/commits/589cdba91ca4c8e80cc6c3c2094315d9a8239d43)

**Comment by ChanhyukPark-Tech** [comment]
> It was by intention that the source code, test code, and documentation are put in the same directory. I thought they will always be edited together, so putting them as near as possible would be a better choice. (If they are far apart, we may forget to edit some files. I wish all docs and test cases are up to date)
> 
> But I agree that it might look messy as more files are added. For that case, what about arranging files like this?
> 
> ```
> is-age/
>   index.ts
>   index.spec.ts
>   index.en.md
>   index.ko.md
> is-birth-date-6/
>   index.ts
>   index.spec.ts
>   index.en.md
>   index.ko.md
> ```

thank you for good suggestion!!

This is a really good method, but...
There are many cases when I developing I find  files by looking for Cmd+P on vscode. Would it be easy to find files if I implemented it like this? 

image

I think it's okay!
If others agree, I think it would be good to divide folders by domain and put doc files(en,ko), test codes, and actual files in them!

There are all the files in one domain, so it's neat as you suggested

**Comment by raon0211** [comment]
Yes, I think we might try this. But I think merging the pull request as is is not bad, so I will merge it right now. Feel free to open another pull request!

**Review by raon0211 — APPROVED** [review]

**Comment by ChanhyukPark-Tech** [comment]
Thx!
Have a good day

**Pull request merged**